source/slang/core.meta.slang

```diff
@@ -727,6 +727,18 @@ __generic<T, let N : int> __intrinsic_op(select) vector<T,N> operator?:(vector<b
 __generic<T> __intrinsic_op(select) T select(bool condition, T ifTrue, T ifFalse);
 __generic<T, let N : int> __intrinsic_op(select) vector<T,N> select(vector<bool,N> condition, vector<T,N> ifTrue, vector<T,N> ifFalse);
 
+[ForceInline]
+__generic<T> __intrinsic_op(select) Optional<T> select(bool condition, __none_t ifTrue, T ifFalse)
```

Comment:
functions with `__intrinsic_op` do not need to have a body, we just need put `;` after the signature here.

Reply:
@csyonghe but select doesn't have a intrinsic overload for none type, does it? I should I might have to enter in the intrinsic part because select is an intrinsic.

Reply:
Alright, I've removed it to reflect that.